tests/basic/customer_issues_test.go

```diff
@@ -757,3 +757,185 @@ var _ = Describe("{CreateCloudSnapAndDelete}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+func getVolumeAttachedNodeAndKill(volName *volume.Volume) (string, *node.Node, error) {
```

Comment:
can you just return node as it has node ip/name details also

Reply:
done